easyprivacy/easyprivacy_general.txt

```diff
@@ -3046,6 +3046,8 @@
 /no-impression.gif?
 /nofastat.
 /notifications/analytics
+/noui/eventlogger.js
+/noui/StatProvider.js
```

Comment:
I don't see these today...

<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +226 | \|\|googletagmanager.com^ | -- | thumbnails.porncore.net | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-3598761-4` |
| +226 | \|\|porncore.net/*.php | -- | thumbnails.porncore.net | 1 | script | `http://thumbnails.porncore.net/jgkmraadutuszx.php` |
| +52 | \|\|googletagmanager.com^ | -- | thumbnails.porncore.net | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-3598761-4` |
| +52 | \|\|porncore.net/*.php | -- | thumbnails.porncore.net | 1 | script | `http://thumbnails.porncore.net/jgkmraadutuszx.php` |
| +0 | \|\|googletagmanager.com^ | -- | thumbnails.porncore.net | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-3598761-4` |
| +0 | \|\|porncore.net/*.php | -- | thumbnails.porncore.net | 1 | script | `http://thumbnails.porncore.net/jgkmraadutuszx.php` |
</details>

Reply:
<details>

![porncore](https://user-images.githubusercontent.com/58900598/109731411-8b112b80-7bfe-11eb-89dc-1e25f80cd755.png)

</details>

Reply:
Cool, you even got another domain (`pc20160301.com`), witch I didn't see at all :smile: 

Chose I saw was 

```
31.media.tumblr.com
40.media.tumblr.com
64.media.tumblr.com
69.media.tumblr.com
assets.tumblr.com
counter.yadro.ru
pc180101.com
pinduck.com
thebestfreeporn.net
www.googletagmanager.com
www.redditstatic.com
```

Thanks for your follow up :+1: